tests/e2e/Makefile

```diff
@@ -18,9 +18,27 @@ include $(CURDIR)/versions.mk
 
 E2E_RUNTIME ?= docker
 
+E2E_INSTALL_CTK ?= true
+
+E2E_IMAGE_NAME ?= ghcr.io/nvidia/container-toolkit
+E2E_IMAGE_TAG ?=
+ifeq ($(E2E_IMAGE_TAG),)
+$(error E2E_IMAGE_TAG is not set)
+endif
```

Comment:
Would it not make sense to use the tag that would be generated when running:
```
make -f deployments/container/Makefile build-ubuntu20.04
```
in the root by default.

i.e:
```
        --tag nvidia/container-toolkit:1.17.3-ubuntu20.04 \
```

Reply:
I just coppied a snippet from the shell output when running the make command. Do we think the following is a reasonable mode of operation:

1. Run `./scripts/build-packages.sh ubuntu18.04-amd64` to build the packages.
2. Run `make -f deployment/container/Makefile build` to build the required image.
3. Run `make -f tests/e2e/Makefile test` to run the tests for the built version locally?